gradle.properties

```diff
@@ -60,6 +60,7 @@ dependencyLicenseReportVersion=2.1
 
 jaxbApiVersion=2.3.1
 jjwtVersion=0.11.5
+graalvmJsVersion=23.0.2
```

Comment:
Any reason why this can't be `23.1.1`?

Reply:
No great reason: I wanted to keep the graalVersion the same for both dependencies. And [org.graal.js:js](https://mvnrepository.com/artifact/org.graalvm.js/js) doesnt have `23.1.1` yet